public_html/css/main.scss

```diff
@@ -1359,6 +1368,10 @@ a.list-group-item {
   }
 }
 
+#formPreview {
+  .controls select { margin-bottom:20px; }
+}
+
```

Comment:
Does this need to be included with the public/oai checkbox update? Nothing shows different in the form preview when these buttons are selected. If it isn't relevant please revert.